/rove_controller_node:
  ros__parameters:
  # Buttons
    A: 0
    B: 1
    X: 2
    Y: 3
    LB: 4
    RB: 5
    VIEW: 6
    MENU: 7
    XBOX: 8
    LS: 9
    RS: 10
    SHARE: 11
  # Axes
    LS_X: 0
    LS_Y: 1
    LT: 2
    RS_X: 3
    RS_Y: 4
    RT: 5
    D_PAD_X: 6
    D_PAD_Y: 7